deploy/src/erc_to_native/home.js

```diff
@@ -51,6 +50,12 @@ const DEPLOYMENT_ACCOUNT_ADDRESS = privateKeyToAddress(DEPLOYMENT_ACCOUNT_PRIVAT
 const isRewardableBridge = HOME_REWARDABLE === 'BOTH_DIRECTIONS'
 const isFeeManagerPOSDAO = HOME_FEE_MANAGER_TYPE === 'POSDAO_REWARD'
 
+let VALIDATORS_REWARD_ACCOUNTS = []
+
+if (isRewardableBridge) {
```

Comment:
In case of POSDAO Fee Manager we also don't need to specify validators reward accounts since the reward will be distributed among the network validators and stakers. The same could be applicable for `erc-to-erc` mode.

Reply:
Added special checks for POSDAO Fee Manager https://github.com/poanetwork/poa-bridge-contracts/pull/272/commits/3d892ed69169ee8737d66116940ef896740b49e3